Automattic/wp-calypso
# Jetpack Manage: Add Stripe information in the 'Add payment method' page.

This pull request adds information for the Stripe payment provider to the 'Add Payment Method' page. This provides confidence that card information is managed by Stripe, which has a bigger name in the Payment provider industry.

<img width="1398" alt="Screen Shot 2024-01-11 at 5 40 41 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/7ca2190d-0021-45f4-8558-85301b989103">


Depends on https://github.com/Automattic/wp-calypso/pull/86276
Closes https://github.com/Automattic/jetpack-genesis/issues/181

## Proposed Changes

* Add the `PaymentMethodStripeInfo` component to the page.


## Testing Instructions
Since these changes are made specifically for agencies, you must set yourself(partner) as an agency - 2c49b-pb.

* Use the Jetpack cloud live link below and go to the new Add Payment Method page (`/partner-portal/payment-methods/add?flags=jetpack/card-addition-improvements`)
* Confirm that the Stripe payment provider information is visible on the side.
* Also test that the UI is responsive and works on different screen sizes.

## Pre-merge Checklist
